docs/App Service Provider Guide/index.md

```diff
@@ -126,3 +126,96 @@ With regards to permission handling during RPC passing:
 
 * For RPCs which are known to Core (determined by its RPC spec version), they are checked normally against the policy table. As such, the ASP can assume in this case that the app specifically has permissions to use the this RPC in its current HMI level.
 * For RPCs unknown to Core, an ASC needs to be granted specific permissions by the OEM (controlled by the `allow_unknown_rpc_passthrough` policy field) to send this message, even if it is handled by the ASP.
+
+## Example Use Case
+
+### Sending a POI to a Navigation Provider
+
+Before the implementation of App Services, SDL applications could only send points of interest to the vehicle's embedded navigation by using the `SendLocation` RPC. The App Services feature was created as a generalized solution to answer the problem question: "How can my SDL application send a point of interest to the active SDL navigation application?". 
+
+Through RPC Passing, the App Services feature allows a non-navigation SDL application (App Service Consumers) to send a `SendLocation` RPC request to SDL Core. If there is an active navigation service, SDL Core will route the request to the SDL navigation application (App Service provider) instead of the vehicle's navigation system.
```

Comment:
```suggestion
Through RPC Passing, the App Services feature allows a non-navigation SDL application (App Service Consumer) to send a `SendLocation` RPC request to SDL Core. If there is an active navigation service, SDL Core will route the request to the SDL navigation application (App Service Provider) instead of the vehicle's navigation system.
```